pages/rss.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { NextPageContext } from "next";
+import { title } from "case";
+
+import { getInitialBlogPosts, Post } from "../util/getInitialBlogPosts";
+
+const getRssXml = (blogPosts: Post[]) => {
+  let rssItemsXml = "";
+  blogPosts.forEach(post => {
```

Comment:
Can we assign `blogPosts.map` to a `const` instead? Its control flow is easier to track for newcomers.

Reply:
Done